exec/java-exec/src/test/java/org/apache/drill/TestFunctionsWithTypeExpoQueries.java

```diff
@@ -213,7 +221,7 @@ public void tesIsNull() throws Exception {
     List<Pair<SchemaPath, TypeProtos.MajorType>> expectedSchema = Lists.newArrayList();
     TypeProtos.MajorType majorType = TypeProtos.MajorType.newBuilder()
         .setMinorType(TypeProtos.MinorType.BIT)
-        .setMode(TypeProtos.DataMode.REQUIRED)
+        .setMode(TypeProtos.DataMode.OPTIONAL)
```

Comment:
why do you have to change the expected result for this query? Should  "r_name is null" be a required type?